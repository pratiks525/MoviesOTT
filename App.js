/**
 * Sample React Native App
 * https://github.com/facebook/react-native
 *
 * @format
 */

import React from 'react';

import { SafeAreaView, StyleSheet } from 'react-native';
import Dashboard from './src/screens/dashboard';
import { BLACK } from './src/theme/colors';

function App() {
  return (
    <SafeAreaView style={style.background}>
      <Dashboard />
    </SafeAreaView>
  );
}

const style = StyleSheet.create({
  background : {
      backgroundColor: BLACK,
      flex: 1
  }
})

export default App;
