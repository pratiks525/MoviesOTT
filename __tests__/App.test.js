import React from 'react';
import { render } from '@testing-library/react-native';
import App from '../App';

describe('test the app folder', ()=>{
  it('check app container', ()=>{
    const container = render(<App/>);
    expect(container).toMatchSnapshot();
  })
})  