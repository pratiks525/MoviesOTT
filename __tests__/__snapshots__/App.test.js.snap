// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test the app folder check app container 1`] = `
<RCTSafeAreaView
  style={
    {
      "backgroundColor": "#000000",
      "flex": 1,
    }
  }
>
  <RCTSafeAreaView>
    <View
      accessibilityIgnoresInvertColors={true}
    >
      <Image
        className="sc-eDLKkx keSWsE"
        source={
          {
            "testUri": "../../../src/assets/images/nav_bar.png",
          }
        }
        style={
          [
            {
              "bottom": 0,
              "left": 0,
              "position": "absolute",
              "right": 0,
              "top": 0,
            },
            {
              "height": undefined,
              "width": undefined,
            },
            undefined,
          ]
        }
      />
      <View
        className="sc-dmyCSP gSHtIV"
      >
        <View
          className="sc-hLQSwg dsyzTX"
        >
          <View
            accessibilityValue={
              {
                "max": undefined,
                "min": undefined,
                "now": undefined,
                "text": undefined,
              }
            }
            accessible={true}
            focusable={true}
            onClick={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            testID="back-icon"
          >
            <Image
              className="sc-fsYfdN fViPom"
              source={
                {
                  "testUri": "../../../src/assets/images/back.png",
                }
              }
            />
          </View>
          <Text
            className="sc-jTQCzO dduKOR"
          >
            Romantic Comedy
          </Text>
        </View>
        <View
          accessibilityValue={
            {
              "max": undefined,
              "min": undefined,
              "now": undefined,
              "text": undefined,
            }
          }
          accessible={true}
          focusable={true}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          testID="search-icon"
        >
          <Image
            className="sc-fsYfdN fViPom"
            source={
              {
                "testUri": "../../../src/assets/images/search.png",
              }
            }
          />
        </View>
      </View>
    </View>
    <RCTScrollView
      ListEmptyComponent={[Function]}
      contentContainerStyle={
        {
          "marginTop": 40,
          "padding": 7,
          "paddingBottom": 40,
        }
      }
      data={
        [
          {
            "name": "The Birds",
            "poster-image": "poster1.jpg",
          },
          {
            "name": "Rear Window",
            "poster-image": "poster2.jpg",
          },
          {
            "name": "Family Pot",
            "poster-image": "poster3.jpg",
          },
          {
            "name": "Family Pot",
            "poster-image": "poster2.jpg",
          },
          {
            "name": "Rear Window",
            "poster-image": "poster1.jpg",
          },
          {
            "name": "The Birds",
            "poster-image": "poster3.jpg",
          },
          {
            "name": "Rear Window",
            "poster-image": "poster3.jpg",
          },
          {
            "name": "The Birds",
            "poster-image": "poster2.jpg",
          },
          {
            "name": "Family Pot",
            "poster-image": "poster1.jpg",
          },
          {
            "name": "The Birds",
            "poster-image": "poster1.jpg",
          },
          {
            "name": "The Birds",
            "poster-image": "poster1.jpg",
          },
          {
            "name": "Rear Window",
            "poster-image": "poster2.jpg",
          },
          {
            "name": "Family Pot",
            "poster-image": "poster3.jpg",
          },
          {
            "name": "Family Pot",
            "poster-image": "poster2.jpg",
          },
          {
            "name": "Rear Window",
            "poster-image": "poster1.jpg",
          },
          {
            "name": "The Birds",
            "poster-image": "poster3.jpg",
          },
          {
            "name": "Rear Window",
            "poster-image": "poster3.jpg",
          },
          {
            "name": "The Birds",
            "poster-image": "poster2.jpg",
          },
          {
            "name": "Family Pot",
            "poster-image": "poster1.jpg",
          },
          {
            "name": "The Birds",
            "poster-image": "poster1.jpg",
          },
        ]
      }
      getItem={[Function]}
      getItemCount={[Function]}
      keyExtractor={[Function]}
      keyboardShouldPersistTaps="always"
      onContentSizeChange={[Function]}
      onEndReached={[Function]}
      onEndReachedThreshold={0.5}
      onLayout={[Function]}
      onMomentumScrollBegin={[Function]}
      onMomentumScrollEnd={[Function]}
      onScroll={[Function]}
      onScrollBeginDrag={[Function]}
      onScrollEndDrag={[Function]}
      removeClippedSubviews={false}
      renderItem={[Function]}
      scrollEventThrottle={0.0001}
      showsVerticalScrollIndicator={false}
      stickyHeaderIndices={[]}
      testID="movie-list"
      viewabilityConfigCallbackPairs={[]}
    >
      <View>
        <View
          onFocusCapture={[Function]}
          onLayout={[Function]}
          style={null}
        >
          <View
            style={
              {
                "flexDirection": "row",
              }
            }
          >
            <View
              className="sc-guDLey iFHBPQ"
            >
              <Image
                onError={[Function]}
                source={
                  {
                    "testUri": "../../../src/assets/images/poster1.jpg",
                  }
                }
                style={
                  {
                    "height": 347,
                    "width": 231.33333333333334,
                  }
                }
                testID="thumbnail-image"
              />
              <Text
                className="sc-beySPh fDxfuz"
                numberOfLines={1}
              >
                The Birds
              </Text>
            </View>
            <View
              className="sc-guDLey iFHBPQ"
            >
              <Image
                onError={[Function]}
                source={
                  {
                    "testUri": "../../../src/assets/images/poster2.jpg",
                  }
                }
                style={
                  {
                    "height": 347,
                    "width": 231.33333333333334,
                  }
                }
                testID="thumbnail-image"
              />
              <Text
                className="sc-beySPh fDxfuz"
                numberOfLines={1}
              >
                Rear Window
              </Text>
            </View>
            <View
              className="sc-guDLey iFHBPQ"
            >
              <Image
                onError={[Function]}
                source={
                  {
                    "testUri": "../../../src/assets/images/poster3.jpg",
                  }
                }
                style={
                  {
                    "height": 347,
                    "width": 231.33333333333334,
                  }
                }
                testID="thumbnail-image"
              />
              <Text
                className="sc-beySPh fDxfuz"
                numberOfLines={1}
              >
                Family Pot
              </Text>
            </View>
          </View>
        </View>
        <View
          onFocusCapture={[Function]}
          onLayout={[Function]}
          style={null}
        >
          <View
            style={
              {
                "flexDirection": "row",
              }
            }
          >
            <View
              className="sc-guDLey iFHBPQ"
            >
              <Image
                onError={[Function]}
                source={
                  {
                    "testUri": "../../../src/assets/images/poster2.jpg",
                  }
                }
                style={
                  {
                    "height": 347,
                    "width": 231.33333333333334,
                  }
                }
                testID="thumbnail-image"
              />
              <Text
                className="sc-beySPh fDxfuz"
                numberOfLines={1}
              >
                Family Pot
              </Text>
            </View>
            <View
              className="sc-guDLey iFHBPQ"
            >
              <Image
                onError={[Function]}
                source={
                  {
                    "testUri": "../../../src/assets/images/poster1.jpg",
                  }
                }
                style={
                  {
                    "height": 347,
                    "width": 231.33333333333334,
                  }
                }
                testID="thumbnail-image"
              />
              <Text
                className="sc-beySPh fDxfuz"
                numberOfLines={1}
              >
                Rear Window
              </Text>
            </View>
            <View
              className="sc-guDLey iFHBPQ"
            >
              <Image
                onError={[Function]}
                source={
                  {
                    "testUri": "../../../src/assets/images/poster3.jpg",
                  }
                }
                style={
                  {
                    "height": 347,
                    "width": 231.33333333333334,
                  }
                }
                testID="thumbnail-image"
              />
              <Text
                className="sc-beySPh fDxfuz"
                numberOfLines={1}
              >
                The Birds
              </Text>
            </View>
          </View>
        </View>
        <View
          onFocusCapture={[Function]}
          onLayout={[Function]}
          style={null}
        >
          <View
            style={
              {
                "flexDirection": "row",
              }
            }
          >
            <View
              className="sc-guDLey iFHBPQ"
            >
              <Image
                onError={[Function]}
                source={
                  {
                    "testUri": "../../../src/assets/images/poster3.jpg",
                  }
                }
                style={
                  {
                    "height": 347,
                    "width": 231.33333333333334,
                  }
                }
                testID="thumbnail-image"
              />
              <Text
                className="sc-beySPh fDxfuz"
                numberOfLines={1}
              >
                Rear Window
              </Text>
            </View>
            <View
              className="sc-guDLey iFHBPQ"
            >
              <Image
                onError={[Function]}
                source={
                  {
                    "testUri": "../../../src/assets/images/poster2.jpg",
                  }
                }
                style={
                  {
                    "height": 347,
                    "width": 231.33333333333334,
                  }
                }
                testID="thumbnail-image"
              />
              <Text
                className="sc-beySPh fDxfuz"
                numberOfLines={1}
              >
                The Birds
              </Text>
            </View>
            <View
              className="sc-guDLey iFHBPQ"
            >
              <Image
                onError={[Function]}
                source={
                  {
                    "testUri": "../../../src/assets/images/poster1.jpg",
                  }
                }
                style={
                  {
                    "height": 347,
                    "width": 231.33333333333334,
                  }
                }
                testID="thumbnail-image"
              />
              <Text
                className="sc-beySPh fDxfuz"
                numberOfLines={1}
              >
                Family Pot
              </Text>
            </View>
          </View>
        </View>
        <View
          onFocusCapture={[Function]}
          onLayout={[Function]}
          style={null}
        >
          <View
            style={
              {
                "flexDirection": "row",
              }
            }
          >
            <View
              className="sc-guDLey iFHBPQ"
            >
              <Image
                onError={[Function]}
                source={
                  {
                    "testUri": "../../../src/assets/images/poster1.jpg",
                  }
                }
                style={
                  {
                    "height": 347,
                    "width": 231.33333333333334,
                  }
                }
                testID="thumbnail-image"
              />
              <Text
                className="sc-beySPh fDxfuz"
                numberOfLines={1}
              >
                The Birds
              </Text>
            </View>
            <View
              className="sc-guDLey iFHBPQ"
            >
              <Image
                onError={[Function]}
                source={
                  {
                    "testUri": "../../../src/assets/images/poster1.jpg",
                  }
                }
                style={
                  {
                    "height": 347,
                    "width": 231.33333333333334,
                  }
                }
                testID="thumbnail-image"
              />
              <Text
                className="sc-beySPh fDxfuz"
                numberOfLines={1}
              >
                The Birds
              </Text>
            </View>
            <View
              className="sc-guDLey iFHBPQ"
            >
              <Image
                onError={[Function]}
                source={
                  {
                    "testUri": "../../../src/assets/images/poster2.jpg",
                  }
                }
                style={
                  {
                    "height": 347,
                    "width": 231.33333333333334,
                  }
                }
                testID="thumbnail-image"
              />
              <Text
                className="sc-beySPh fDxfuz"
                numberOfLines={1}
              >
                Rear Window
              </Text>
            </View>
          </View>
        </View>
        <View
          onFocusCapture={[Function]}
          onLayout={[Function]}
          style={null}
        >
          <View
            style={
              {
                "flexDirection": "row",
              }
            }
          >
            <View
              className="sc-guDLey iFHBPQ"
            >
              <Image
                onError={[Function]}
                source={
                  {
                    "testUri": "../../../src/assets/images/poster3.jpg",
                  }
                }
                style={
                  {
                    "height": 347,
                    "width": 231.33333333333334,
                  }
                }
                testID="thumbnail-image"
              />
              <Text
                className="sc-beySPh fDxfuz"
                numberOfLines={1}
              >
                Family Pot
              </Text>
            </View>
            <View
              className="sc-guDLey iFHBPQ"
            >
              <Image
                onError={[Function]}
                source={
                  {
                    "testUri": "../../../src/assets/images/poster2.jpg",
                  }
                }
                style={
                  {
                    "height": 347,
                    "width": 231.33333333333334,
                  }
                }
                testID="thumbnail-image"
              />
              <Text
                className="sc-beySPh fDxfuz"
                numberOfLines={1}
              >
                Family Pot
              </Text>
            </View>
            <View
              className="sc-guDLey iFHBPQ"
            >
              <Image
                onError={[Function]}
                source={
                  {
                    "testUri": "../../../src/assets/images/poster1.jpg",
                  }
                }
                style={
                  {
                    "height": 347,
                    "width": 231.33333333333334,
                  }
                }
                testID="thumbnail-image"
              />
              <Text
                className="sc-beySPh fDxfuz"
                numberOfLines={1}
              >
                Rear Window
              </Text>
            </View>
          </View>
        </View>
        <View
          onFocusCapture={[Function]}
          onLayout={[Function]}
          style={null}
        >
          <View
            style={
              {
                "flexDirection": "row",
              }
            }
          >
            <View
              className="sc-guDLey iFHBPQ"
            >
              <Image
                onError={[Function]}
                source={
                  {
                    "testUri": "../../../src/assets/images/poster3.jpg",
                  }
                }
                style={
                  {
                    "height": 347,
                    "width": 231.33333333333334,
                  }
                }
                testID="thumbnail-image"
              />
              <Text
                className="sc-beySPh fDxfuz"
                numberOfLines={1}
              >
                The Birds
              </Text>
            </View>
            <View
              className="sc-guDLey iFHBPQ"
            >
              <Image
                onError={[Function]}
                source={
                  {
                    "testUri": "../../../src/assets/images/poster3.jpg",
                  }
                }
                style={
                  {
                    "height": 347,
                    "width": 231.33333333333334,
                  }
                }
                testID="thumbnail-image"
              />
              <Text
                className="sc-beySPh fDxfuz"
                numberOfLines={1}
              >
                Rear Window
              </Text>
            </View>
            <View
              className="sc-guDLey iFHBPQ"
            >
              <Image
                onError={[Function]}
                source={
                  {
                    "testUri": "../../../src/assets/images/poster2.jpg",
                  }
                }
                style={
                  {
                    "height": 347,
                    "width": 231.33333333333334,
                  }
                }
                testID="thumbnail-image"
              />
              <Text
                className="sc-beySPh fDxfuz"
                numberOfLines={1}
              >
                The Birds
              </Text>
            </View>
          </View>
        </View>
        <View
          onFocusCapture={[Function]}
          onLayout={[Function]}
          style={null}
        >
          <View
            style={
              {
                "flexDirection": "row",
              }
            }
          >
            <View
              className="sc-guDLey iFHBPQ"
            >
              <Image
                onError={[Function]}
                source={
                  {
                    "testUri": "../../../src/assets/images/poster1.jpg",
                  }
                }
                style={
                  {
                    "height": 347,
                    "width": 231.33333333333334,
                  }
                }
                testID="thumbnail-image"
              />
              <Text
                className="sc-beySPh fDxfuz"
                numberOfLines={1}
              >
                Family Pot
              </Text>
            </View>
            <View
              className="sc-guDLey iFHBPQ"
            >
              <Image
                onError={[Function]}
                source={
                  {
                    "testUri": "../../../src/assets/images/poster1.jpg",
                  }
                }
                style={
                  {
                    "height": 347,
                    "width": 231.33333333333334,
                  }
                }
                testID="thumbnail-image"
              />
              <Text
                className="sc-beySPh fDxfuz"
                numberOfLines={1}
              >
                The Birds
              </Text>
            </View>
          </View>
        </View>
      </View>
    </RCTScrollView>
  </RCTSafeAreaView>
</RCTSafeAreaView>
`;
